packages/core/src/domain/telemetry/telemetry.ts

```diff
@@ -80,6 +80,7 @@ export function startTelemetry(configuration: Configuration): Telemetry {
           format_version: 2 as const,
         },
         telemetry: event as any, // https://github.com/microsoft/TypeScript/issues/48457
+        experimental_features: getExperimentalFeatures(),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏should not we put that on the schema?